<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255,255,255)</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Главная</string>
    </attribute>
    <widget class="QPushButton" name="chooseButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(63, 114, 175);
color:rgb(255,255,255);
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string>Выбрать файл</string>
     </property>
    </widget>
    <widget class="QLabel" name="nametest_text">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Укажите название теста:</string>
     </property>
    </widget>
    <widget class="QLabel" name="discipline_text">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выберите дисциплину:</string>
     </property>
    </widget>
    <widget class="QLabel" name="quantity_text">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Укажите количество вопросов:</string>
     </property>
    </widget>
    <widget class="QLabel" name="time_text">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Укажите время тестирования, в мин:</string>
     </property>
    </widget>
    <widget class="QLabel" name="maxscore_text">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Укажите максимальный балл за 1 вопрос:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="calculateButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>390</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(63, 114, 175);
color:rgb(255,255,255);
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string>Расчитать</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="nametest">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(243, 243, 243);
border-radius: 5px;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="quantity">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(243, 243, 243);
border-radius: 5px;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="time_box">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>210</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(243, 243, 243);
border-radius: 5px;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="maxscore">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>260</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(243, 243, 243);
border-radius: 5px;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="discipline_box">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>110</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(243, 243, 243);
border-radius: 5px;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="file">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(243, 243, 243);
border-radius: 5px;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Тесты</string>
    </attribute>
    <widget class="QPushButton" name="addButton_2">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>30</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(63, 114, 175);
color:rgb(255,255,255);
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>561</width>
       <height>411</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>559</width>
        <height>409</height>
       </rect>
      </property>
      <widget class="QTableWidget" name="tableWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>561</width>
         <height>411</height>
        </rect>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>46</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>185</number>
       </attribute>
       <column>
        <property name="text">
         <string>Название документа</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Новый столбец</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Дисциплина</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="downloadButton_2">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>80</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(63, 114, 175);
color:rgb(255,255,255);
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string>Скачать</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteButton_2">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>130</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(63, 114, 175);
color:rgb(255,255,255);
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
